### ISSUE TITLE
How to reuse precompiled modules?

### ISSUE THREAD

**Issue by wollmers** [yuki-kimoto/SPVM#210]
Not sure, if this is an issue of not working (Mac OS 10.15) or a documentation issue.

What I did:

- following the tutorial https://metacpan.org/release/KIMOTO/SPVM-0.9116/view/lib/SPVM/Document/Tutorial.pm and using the example "Precompiled SPVM Method."

- setting Environment Variable "SPVM_BUILD_DIR":

```
export SPVM_BUILD_DIR=~/.spvm_build
``` 

It builds and compiles, the files could be found under `~/.spvm_build`, but the script using the module compiles it each time and doesn't reuse it. I would expect that SPVM recognises successive uses automagically. Or it is missing in the documentation, what has to be done or configured to install and reuse the precompiled modules.

**Comment by yuki-kimoto** [comment]
Hi, @wollmers .

> It builds and compiles, the files could be found under , but the script using the module compiles it each time and doesn't reuse it.

If the module compiles it each time, it is bug.

I will try and fix today.

**Comment by yuki-kimoto** [comment]
I try the example.

```
SPVM_CC_DEBUG=1 perl sum_precompile.pl
```

Output:

```
cc -shared -O2 -o /home/kimoto/.spvm_build/work/lib/SPVM/MyMath.precompile.so 
```

the script using the module *doesn't* compiles it each time

the script using the module *links* it each time.

This will be improved. I try to fix it.

**Comment by yuki-kimoto** [comment]
I fixed this at SPVM-0.9120.

**Comment by wollmers** [comment]
@yuki-kimoto 
> I fixed this at SPVM-0.9120.

Thanks for prompt reaction. Still saw your fix and will test it, if it's on CPAN.

**Comment by wollmers** [comment]
Works now in SPVM-0.9120.
